src/foam/nanos/auth/CachingAuthService.java

```diff
@@ -87,7 +98,7 @@ public CachingAuthService(AuthService delegate) {
   @Override
   public boolean check(foam.core.X x, String permission) {
     if ( x == null || permission == null ) return false;
-
+    if ( ((User)x.get("user")).getId() == 1 ) return true;
```

Comment:
To save a context lookup (not sure if worthwhile or not), we could do the lookup here and pass the result as an argument to `getPermissionMap` so it doesn't need to do the same lookup.

Reply:
getPermissionMap is also called by a static method, so I don't think it is worth it. You can do millions of context lookups per second anyway.